reviewrot/basereview.py

```diff
@@ -318,13 +318,8 @@ def _format_json(self, i, N, show_last_comment):
             formatted_string(str): Formatted string as per style
         """
         # Include a comma after every entry, except the last.
-        if i and N:
-            suffix = ',' if i < N - 1 else ''
-
-            return json.dumps(self.__json__(show_last_comment), indent=2) + suffix
-        # If we omit the return statement, this function will return None
-        # which will cause a malformed json.
-        return ''
+        suffix = ',' if i < N - 1 else ''
```

Comment:
It's a bit odd that the `format` method has defaults for `i` and `N` set to `None` and this method wouldn't handle them being `None`, but since there's no code path where it would actually occur, I guess it's ok. I think it would be nicer if the arguments were mandatory.

Reply:
Let's make it so.  I'll re-submit with that change and request another review.

Reply:
See f03e2ba.